src/PizzaConstructor.js

```diff
@@ -0,0 +1,286 @@
+import React, { useState } from 'react';
+import { calcTotalPrice } from './calcTotalPrice';
+
+const PizzaConstructor = () => {
+  // Default pizza - thin tomato
+  const [order, setOrder] = useState({
+    size: 30,
+    base: 'thin',
+    sauce: 'tomato sauce',
+    cheese: [],
+    veg: [],
+    meat: [],
+  });
+
+  const [orderList, setOrderList] = useState([]);
+
+  // Total Order Price
+  const totalPrice = calcTotalPrice(order);
+
+  function showOrder(ev) {
+    ev.preventDefault();
+    const orderArr = [].concat(...Object.values(order));
```

Comment:
```suggestion
    const orderArr = [...Object.values(order)];
```
Разве так не будет работать? Как по мне более коротко

Reply:
+ я бы убрал с имени переменной слово Arr. Для массивов обычно дают имя существительного в мнодественной форме, тогда и так ясно, что это массив без слова arr

Например `orders`

Reply:
> Разве так не будет работать? Как по мне более коротко

так тоже работает, но в этом случае к заказу добавляется пустой element inside и тогда это выглядит как:
`30, thin, tomato sauce, , tomato`